test_config/good_for_refl/refl/config_trunc.py

```diff
@@ -0,0 +1,33 @@
+"""
+FOR TESTING
+
+Valid configuration script for a reflectometry beamline
+"""
+import random
+import string
+from ReflectometryServer import *
+
+
+RANDOM_STRING = ''.join(random.choices(string.ascii_letters + string.digits, k=5000))
```

Comment:
Please seed the random number generator if you absolutely must use `random`, to ensure we don't get random failures that are very hard to debug